Fix screen points display (#862)
pascalaubry/papi-web: RELEASE_NOTES.md

- Ajout de la possibilité de ne pas afficher les adversaires sur les écrans d'appariements par ordre alphabétique (2.8.0)
- Correction de la suppression des résultats (2.8.2)

> - Correction de l'affichage des points lors de l'édition des appariements via l'interface d'administration (2.8.4)

Reviewer: ```suggestion
- Correction de l'affichage des points sur les écrans (2.8.4)
```

- Added the possibility not to show the opponents on pairings screens by alphabetical order (2.8.0)
- Fixed deletion of results (2.8.2)

> - Fixed the point display when editing pairings via the admin interface (2.8.4)

Reviewer: ```suggestion
- Fixed the display of points on screens (2.8.4)
```
